zyte_parsers/utils.py

```diff
@@ -55,19 +56,22 @@ def strip_urljoin(base_url: Optional[str], url: Optional[str]) -> str:
     return urljoin(base_url or "", url or "")
 
 
-def extract_link(a_node: SelectorOrElement, base_url: str) -> Optional[str]:
+def add_https_to_url(url: str) -> str:
+    if url.startswith(("http://", "https://")):
+        return url
+
+    parsed_url = urlparse(url)
+    if not parsed_url.scheme and parsed_url.netloc:
+        parsed_url = parsed_url._replace(scheme="https")
+
+    return str(urlunparse(parsed_url))
+
+
+def extract_link(
+    a_node: SelectorOrElement, base_url: str, force_safe: bool = False
```

Comment:
As I understand you're adding `force_safe` to make new behavior optional, I wonder if it's possible to apply that unconditionally to keep API simpler, or there are any cases where you are worried it can break working code?